src/operators/transformer.operators.ts

```diff
@@ -15,7 +31,7 @@ export function omitTransformerProperties<T, F extends keyof T>(
     // decorate each omitted field as `@Exclude()`
     for (const field of fields) {
         if (typeof field === 'string') {
-            decorator(cls, field);
+            decorator(cls.prototype, field);
```

Comment:
From [the docs](https://www.typescriptlang.org/docs/handbook/decorators.html#property-decorators)

> The expression for the property decorator will be called as a function at runtime, with the following two arguments:
>  1.  Either the constructor function of the class for a static member, or the prototype of the class for an instance member.
>  2. The name of the member.

It appears that `class-transformer` didn't care, but `class-validator` depends on receiving the prototype.